### PR TITLE
Adding css pattern docs

### PR DESCRIPTION
**Pull request by ellisonbg**
@sccolbert @blink1073

### CODE REVIEW

File: tutorial/css.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+class name that gives a semantic naming of the component, such as:
+
+* `jp-MyWidget-toolbar`
+* `jp-MyWidget-button`
+* `jp-MyWidget-contentButton`
+
+Thus, the general naming of CSS classes is of the form
+`jp-WidgetName-semanticChild`.
+
+Forth, some CSS classes are used to modify the state of a widget:
```
Comment (blink1073) [inline tutorial/css.md]: Fourth*

**Comment by blink1073** [comment]
LGTM modulo my comment above.

**Comment by ellisonbg** [comment]
OK, typo fixed, merge away. I will update these docs as we evolve the approach in other PRs.

**Comment by blink1073** [comment]
Rolling...

**Pull request merged**